cime_config/buildnml

```diff
@@ -675,9 +700,11 @@ else if ($OCN_GRID == tnx1v1 || $OCN_GRID == tnx1v3 || $OCN_GRID == tnx1v4) then
     set BACLIN = 3600.
     set BATROP = 60.
     set CWBDTS = .75e-4
-    set CE     = .5
     set NIWGF  = .4
     set SMTFRC = .false.
+    if ($BLOM_VCOORD == isopyc_bulkml) then
+      set CE     = .5
```

Comment:
Does this mean that CE is undefined if $BLOM_VCOORD /= isopyc_bulkml? Is that ok?

Reply:
CE is first set to a common literature value of 0.06 (Fox-Kemper et al., 2008), then adjusted (quite strongly) if $BLOM_VCOORD == isopyc_bulkml. With hybrid coordinate we found reasonable submesoscale eddy-induced overturning strength with the standard value.